CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Any objection to leaving it at 3.10 for now? That's the cmake version that ships with the current Ubuntu LTS (18.04), and where a lot of developers/users live.

Reply:
Moved to 3.10